src/MongrationDotNet/MongrationDotNet.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
-    <Version>2.1.1</Version>
+    <Version>2.1.2</Version>
   </PropertyGroup>
```

Comment:
Not that it really matters, but when you add a feature (PageThroughAllFilteredDocuments) it should be a minor increase under semver, not a bugfix.

Reply:
Changed